presidio-image-redactor/presidio_image_redactor/bbox.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Tuple
+
+
+class BboxProcessor:
+    """Common module for general bounding box operators."""
+
+    @staticmethod
+    def get_bboxes_from_ocr_results(ocr_results: dict) -> List[dict]:
```

Comment:
There is an assumption here that the user uses tesseract, as `ocr_results` are the tesseract output. If we turn `get_bboxes_from_ocr_results` into a non static method, a user could potentially implement a new `BboxProcessor` and overwrite this method with logic to extract boxes for another OCR engine. WDYT?

Reply:
Good point, I'll remove the `@staticmethod` decorator here.

Though, I do want to point out that the expected format of `ocr_results` here matches what is produced within `ImageAnalyzerEngine.analyze`. Perhaps we should add some sort of schema check inside the `ocr` class that ensures the output of `ocr.perform_ocr` matches the output format from Tesseract?

Reply:
Created an issue to address the OCR output schema checking in the future: https://github.com/microsoft/presidio/issues/1013

Reply:
Thanks, that's a great suggestion.